src/profile/Index.vue

```diff
@@ -1,89 +1,169 @@
+                  <v-list-item-avatar>
+                    <v-img :src="item.avatar"></v-img>
+                  </v-list-item-avatar>
 
-      <v-col cols="12" sm="6" md="4">
-        <DoNotDiscloseCard :dnd="$user.hide"/>
+                  <v-list-item-content>
+                    <v-list-item-title v-html="item.title"></v-list-item-title>
+                    <v-list-item-subtitle v-html="item.secondary"></v-list-item-subtitle>
+                  </v-list-item-content>
+
+                  <v-list-item-avatar>
+                    <v-icon>mdi-launch</v-icon>
+                  </v-list-item-avatar>
+                </v-list-item>
+              </a>
+            </template>
+          </v-card>
+        </v-list>
       </v-col>
     </v-row>
 
-    <v-subheader>{{ $vuetify.lang.t('$vuetify.profile.index.2sv') }}</v-subheader>
-
     <v-row>
-      <v-col cols="12" sm="6" md="4">
-        <TotpCard :meta="mfa.totp"/>
+      <v-col>
+        <h3>Google Workspace</h3>
       </v-col>
+    </v-row>
 
-      <v-col cols="12" sm="6" md="4">
-        <SecurityKeyCard :meta="mfa.webauthn"/>
+    <v-row>
+      <v-col>
+        <h5>Login with your SIL Google account</h5>
       </v-col>
+    </v-row>
 
-      <v-col cols="12" sm="6" md="4">
-        <BackupCodeCard :meta="mfa.backup"/>
+    <v-row class="mt-10">
+      <v-col>
+        <div class="flex row justify-space-around ml-2">
+          <a v-for="(item, index) in googleItems" :key="index" :href="item.url" target="_blank" class="text-decoration-none">
+            <v-img :id="item.label" :src="item.image" max-width="40px" class="mx-auto"/>
+            <label :for="item.label">{{item.label}}</label>
+          </a>
+        </div>
       </v-col>
     </v-row>
   </BasePage>
 </template>
 
 <script>
 import ProfileProgress from './ProfileProgress'
-import PasswordCard from './PasswordCard'
-import PasswordRecoveryCard from './PasswordRecoveryCard'
-import TotpCard from './TotpCard'
-import SecurityKeyCard from './SecurityKeyCard'
-import BackupCodeCard from './BackupCodeCard'
-import DoNotDiscloseCard from './DoNotDiscloseCard'
-import Attribute from './Attribute'
-import { recoveryMethods, retrieve as retrieveMethods} from '@/global/recoveryMethods';
-import { mfa, retrieve as retrieveMfa } from '@/global/mfa';
+import { recoveryMethods, retrieve as retrieveMethods} from '@/global/recoveryMethods'
+import { mfa, retrieve as retrieveMfa } from '@/global/mfa'
+
 
 export default {
   components: {
-    ProfileProgress,
-    PasswordCard,
-    PasswordRecoveryCard,
-    TotpCard,
-    SecurityKeyCard,
-    BackupCodeCard,
-    DoNotDiscloseCard,
-    Attribute,
+    ProfileProgress
   },
   data: () => ({
     alternates: recoveryMethods.alternates,
     mfa,
   }),
   computed: {
-    hasUnverifiedEmails: vm => vm.alternates.some(m => ! m.verified),
+    org: (vm) => vm.$user.email.split('@')[1].split('.')[0],
+    coreItems(vm) {
```

Comment:
It seems like this whole list of items would be better placed into a private skinny repo, specific to a particular instance of this application, rather than putting all of this information into the open source application itself.

Reply:
good idea.

Reply:
What do you think about bringing in specific names, urls and icon urls as a JSON object from a skinney's env and then checking the `item.name` and giving `item.secondary` a value from locales. It would allow for alternative lists, but they would not be supported by locales, which are hard coded.